data-types.md

```diff
@@ -132,8 +132,6 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 | `battery_percent`       | Integer          | Required if Applicable | Percent battery charge of vehicle, expressed between 0 and 100 |
 | `fuel_percent`       | Integer          | Required if Applicable | Percent fuel in vehicle, expressed between 0 and 100 |
 
-(?) What about  other types of sensor data, e.g. accelerometer? Scooter tip sensors?  Should that be telemetry or an event or ... ?
```

Comment:
I think [surface type](https://github.com/openmobilityfoundation/mobility-data-specification/issues/767) should be in telemetry, and [tip overs](https://github.com/openmobilityfoundation/mobility-data-specification/issues/794) and [parking validation](https://github.com/openmobilityfoundation/mobility-data-specification/issues/815) with events/trip metadata.